### PR TITLE
ITL-6 Add sorting functionality

### PR DESCRIPTION
**Pull request by Fekete-Balazs710**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -166,7 +181,63 @@ function filterTodos(searchValue: string) {
 function saveTodo(todo: TodoType, todoTitle: string, todoDescription: string) {
     todo.title = todoTitle;
     todo.description = todoDescription;
+    saveTodosToLocalStorage()
+    console.log('todo saved')
```
Comment (gerebdavid) [inline src/App.vue]: Delete console logs

File: src/components/TodoDisplay.vue [hunk 2]
```diff
@@ -76,4 +77,20 @@ function saveTodo(todo: TodoType, todoTitle: string, todoDescription: string) {
...
+.fade-enter-active{
+  transition: all 0.8s cubic-bezier(0.55, 0, 0.1, 1);
+}
+
+.fade-enter-from,
+.fade-leave-to {
+  opacity: 0;
+  transform: scaleY(0.01) translate(30px, 0);
+}
+
```
Comment (kincsokiss) [inline src/components/TodoDisplay.vue:93]: Please remove the empty lines.

File: src/components/header/Header.vue [hunk 3]
```diff
@@ -49,6 +52,8 @@ defineProps<Props>()
 //Add emit to modify data in parent component
 const emit = defineEmits(['addTodo', 'onFilterTodos', 'sortTodos', 'reverseTodos'])
 
+const whichArrow = ref('up')
```
Comment (kincsokiss) [inline src/components/header/Header.vue]: Can you rename the `whichArrow` variable and the prop of the TodoSport as well to `selectedArrow` or something like this? The `whichArrow` sounds a bit weird for me.

File: src/components/header/Header.vue [hunk 4]
```diff
@@ -63,6 +68,13 @@ function sortTodos(attribute: string) {
 
 function reverseTodos() {
     emit('reverseTodos')
+    if(whichArrow.value === "down")
```
Comment (kincsokiss) [inline src/components/header/Header.vue]: Please change the if..else statement to a ternary.

File: src/components/todo/TodoSort.vue [hunk 5]
```diff
@@ -61,18 +67,22 @@ import { TodoType } from '../../types/TodoType';
...
 interface Props {
   todos: TodoType[]
+  whichArrow: string
 }
 
 defineProps<Props>()
 
 const emit = defineEmits(['sortTodos', 'reverseTodos'])
 
+const active = ref("")
```
Comment (kincsokiss) [inline src/components/todo/TodoSort.vue]: Please rename the `active` variable to a more descriptive name. Ex. `activeFilter`

**Review by kincsokiss — CHANGES_REQUESTED** [review]

**Review by kincsokiss — APPROVED** [review]

**Review by gerebdavid — APPROVED** [review]

**Pull request merged**